### ISSUE TITLE
Mobile: Android: "More" Toolbar Item Isn't Accessible from Unsupported Block Editor

### ISSUE THREAD

**Issue by SiobhyB** [WordPress/gutenberg#33592]
## Description

The "more" toolbar item isn't currently accessible from the Unsupported Block Editor (UBE). Tapping on it shows a flash of the dropdown but it instantly disappears.

## Step-by-step reproduction instructions

_**Precondition:** To follow these steps, you'll first need to create a post or page on the web that contains a block that's unsupported in the apps. At the time of writing, the tag cloud block is an option. You will also need to be working on a WordPress.com or a Jetpack-connected site._ 

* In the Android app, navigate to a post or page containing an unsupported block. 
* Tap the **?** icon to the unsupported block's upper right and then **Edit using web editor**.
* Tap the "more" icon in the editor's toolbar i.e. the last icon in the toolbar's list, it has three dots.
* Observe how the dropdown menu briefly flashes on the screen, but quickly disappears. 

## Expected behaviour

Tapping on the "more" toolbar item should bring up a dropdown menu with different settings that can be used for further customizing the block.

## Actual behaviour

Tapping on the toolbar item shows a flash of the dropdown but it instantly disappears.

## Screenshots or screen recording (optional)

https://user-images.githubusercontent.com/2998162/126494700-5a175c8b-3637-453b-a634-a903e3695764.mov

## Device information
- Device: Pixel 5
- Operating system: Android 11
- WordPress app version: 17.8

**Comment by SiobhyB** [comment]
Noting that, when I initially saw this report, I assumed it was caused by the recent changes I made [here](https://github.com/WordPress/gutenberg/pull/32395). However, I've confirmed this bug exists even if I revert those changes on my local setup. So, it seems this _isn't_ a recent regression. I'll dive further into this.

**Comment by mchowning** [comment]
Just noting that out of curiousity I tried loading up the tag-cloud block in mobile chrome, and didn't see this issue.

**Comment by SiobhyB** [comment]
In an effort to pinpoint exactly when this bug was introduced, I went back to install some older versions of the app. 

I was able to replicate the issue on version 15.4 of the app (available [here](https://github.com/wordpress-mobile/WordPress-Android/releases/tag/15.4)), which was launched on August 10th, 2020. The UBE was introduced in version 15.3, but my sites wouldn't load correctly in that version. 

It's not clear to me, from related PRs or calls for testing, whether the UBE's "more" toolbar item has _ever_ worked on Android. (As a side note, it works with no issues at all in iOS.)

I'll keep digging. 🕵️‍♀️
